docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -483,6 +483,127 @@ The TransactionManager controls the timeout of database queries.
+```
+{
+    "__comment": "FHIR Server configuration",
+    "fhirServer": {
+        ...
+        "persistence": {
+            "factoryClassname": "com.ibm.fhir.persistence.blob.FHIRPersistenceJDBCBlobFactory",
+            "datasources": {
+                "default": {
+                    ...
+                }
+            },
+            "payload": {
+                "default": {
+                    "__comment": "Azure Blob (azurite docker) configuration for storing FHIR resource payload data",
+                    "type": "azure.blob",
+                    "connectionProperties" : {
+                        "connectionString": "your-azure-connection-string",
+                        "containerName": "default-default"
+                    }
+                }
+            }
+        }
+    }
+}
+```
+
+Container names allowed by Azure Blob are more restrictive than LinuxForHealth FHIR Server tenant and datasource names (for example they must be lower case and `_` is not allowed - see the [official documentation](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction#containers) for more details). For this reason, the container name for each tenant and datasource must be specified in the `fhirServer/persistence/payload/connectionProperties/containerName` property.
+
+#### 3.3.2.5 Configure Payload Offloading Per Tenant
+
+In each tenant `fhir-server-config.json` file, add a `fhirServer/persistence/payload` block containing the connection information for each datasource you have defined under `fhirServer/persistence/datasources` (typically there is just one datasource called `default`). 
+
+```
+{
+    "__comment": "LinuxForHealth FHIR Server configuration",
+    "fhirServer": {
+        ...
+        "persistence": {
+            "datasources": {
+                "default": {
+                    ...
+                }
+            },
+            "payload": {
+                "default": {
+                    "__comment": "Azure Blob configuration for storing FHIR resource payload data",
+                    "type": "azure.blob",
+                    "connectionProperties" : {
+                        "connectionString": "your-azure-connection-string",
+                        "containerName": "default-default"
+                    }
+                }
+            }
+        }
+    }
+}
+```
+
+
+#### 3.3.2.6 Create the Container
+
+The container can be created using the Azure Blob service or by running the following command:
+
+```
+java -jar fhir-persistence-blob-app-*-cli.jar \
+    --fhir-config-dir /path/to/wlp/usr/servers/defaultServer \
+    --tenant-id <tenant-id> \
+    --ds-id <ds-id> \
+    --create-container \
+    --no-dry-run
+```
+
+The container name will be read from the tenant's fhir-server-config.json configuration, so it is important to complete that configuration step before running this command.
```

Comment:
not for this PR, but we should consider adding fhir-persistence-blob-app-*-cli.jar alongside the schema jar in our zip installation package.  i think that will put it at /opt/fhir-server/tools in our main image